src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/EventValidationService.java

```diff
@@ -382,9 +361,82 @@ public void validateJudgementsHasJurisdiction(CaseData caseData, boolean partOfM
         return errors;
     }
 
+    public void validateHearingStatusForCaseCloseEvent(CaseData caseData, List<String> errors) {
+        if (CollectionUtils.isEmpty(caseData.getHearingCollection())) {
+            return;
+        }
+
+        for (var currentHearingTypeItem : caseData.getHearingCollection()) {
+            for (var currentDateListedTypeItem : currentHearingTypeItem.getValue().getHearingDateCollection()) {
+                if (HEARING_STATUS_LISTED.equals(currentDateListedTypeItem.getValue().getHearingStatus())) {
+                    errors.add(CLOSING_LISTED_CASE_ERROR);
+                    break;
+                }
+            }
+        }
+
+    }
+
+    public void validateHearingJudgeAllocationForCaseCloseEvent(CaseData caseData, List<String> errors) {
+
+        if (CollectionUtils.isEmpty(caseData.getHearingCollection())) {
+            return;
+        }
+
+        for (var currentHearingTypeItem : caseData.getHearingCollection()) {
+            for (var currentDateListedTypeItem : currentHearingTypeItem.getValue().getHearingDateCollection()) {
+                if (HEARING_STATUS_HEARD.equals(currentDateListedTypeItem.getValue().getHearingStatus())
+                        && currentHearingTypeItem.getValue().getJudge() == null) {
+                    errors.add(CLOSING_HEARD_CASE_WITH_NO_JUDGE_ERROR);
+                    break;
```

Comment:
I think it should be return otherwise you could potentially add multiple copies of the error message